spec/checkPullRequestJobSpec.js

```diff
@@ -478,25 +721,31 @@ describe('Pull Request Job Spec', () => {
     it('should ping server jobs admin', () => {
       expect(github.issues.createComment).toHaveBeenCalled();
       const author = payloadData.payload.pull_request.user.login;
-      const formText = 'server jobs form'.link(
-        'https://goo.gl/forms/XIj00RJ2h5L55XzU2');
+      const formText = (
+        'server jobs form'.link('https://goo.gl/forms/XIj00RJ2h5L55XzU2'));
       const newLineFeed = '<br>';
       const wikiLinkText = (
         'this guide'.link(
           'https://github.com/oppia/oppia/wiki/Running-jobs-in-production' +
           '#submitting-a-pr-with-a-new-job'));
-      const jobRegistryLink = 'job registry'.link(
-        'https://github.com/oppia/oppia/blob/develop/core/jobs_registry.py');
-      const jobNameLink = 'OppiabotContributionsOneOffJob'.link(modifiedExistingJobFileObj.blob_url);
+      const jobRegistryLink = (
+        'job registry'.link(
+          'https://github.com/oppia/oppia/blob/develop/core/jobs_registry.py'));
+      const jobNameLink =
+        'OppiabotContributionsOneOffJob'.link(
+          modifiedExistingJobFileObj.blob_url
+        );
```

Comment:
Modify this to include the parens as was suggested by @vojtechjelinek https://github.com/oppia/oppiabot/pull/148#discussion_r511668220

Reply:
Done